.github/CODEOWNERS

```diff
@@ -7,3 +7,5 @@
 
 # The yoshi-nodejs team is the default owner for nodejs repositories.
 *     @googleapis/yoshi-nodejs @googleapis/ml-apis
+
+/samples/**/*.py                           @gcseh @googleapis/api-iot @googleapis/yoshi-nodejs
```

Comment:
- We try to avoid individuals in CODEOWNERS if we can, and it looks like gcseh is already a member fo @googleapis/api-iot 
- I don't think `*.py` is going to get us where we want to go in this repo :)

Reply:
right, thanks!